app/src/main/res/layout/fragment_intro_slide.xml

```diff
@@ -1,34 +1,30 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="@color/introSlideBackground"
     android:padding="16dp"
-    android:background="@color/introSlideBackground">
+    tools:background="@color/colorPrimaryDark">
 
-    <LinearLayout
+    <TextView
+        android:id="@+id/description_text"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:orientation="vertical"
-        android:layout_gravity="center">
+        android:layout_centerInParent="true"
```

Comment:
This way only description text will be centered, and title will be above it, shifted upwards on its height value